bin/tachyon

```diff
@@ -16,6 +16,7 @@ function printUsage {
   echo -e "  loadufs       \t Load existing files in underlayer filesystem into Tachyon."
   echo -e "  runTest       \t Run a end-to-end test on a Tachyon cluster."
   echo -e "  runTests      \t Run all end-to-end tests on a Tachyon cluster."
+  echo -e "  jCrashTest    \t Test the Master Journal System in a crash scenario"
```

Comment:
This should actually say "journal" instead of just "j". Also, does it make sense to have this new command, or would it fit under runTest?

Nevermind, it looks like it stops existing server and creates a new one. That info should be in the "help" of the journal test.